python/fbprophet/forecaster.py

```diff
@@ -134,6 +146,15 @@ def __init__(
             holidays['ds'] = pd.to_datetime(holidays['ds'])
         self.holidays = holidays
 
+        if append_holidays is not None:
+            if not (
+                    isinstance(append_holidays, str)
+                    or isinstance(append_holidays, list)
+            ):
+                raise ValueError("append_holidays must be a string \
+                                      or a list of strings")
+                self.calculate_holidays = True
+                self.append_holidays = append_holidays
```

Comment:
put `self.append_holidays = append_holidays` outside the loop so that the attribute will exist and be None if there are no append_holidays.